### PR TITLE
Layout

### PR DESCRIPTION
**Pull request by lannisterrr**
Added - Basic layout for the page

### CODE REVIEW

File: src/Styles/searchBar.css [hunk 1]
```diff
@@ -0,0 +1,60 @@
+.search-bar {
+  --size: 4rem;
+  border: 2px solid var(--text-primary);
+  display: flex;
+  border-radius: 999px;
+  height: var(--size);
+  width: var(--size);
+  padding: 3px;
```
Comment (junaidshaikh-js) [inline src/Styles/searchBar.css:8]: Liked the use of local variables

File: src/components/ResponsiveSearchBar.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
```
Comment (junaidshaikh-js) [inline src/components/ResponsiveSearchBar.js:1]: From React V17 you don't need to import React

**Review by junaidshaikh-js — APPROVED** [review]
Good work done! 

Your code is clean and readable, with good use of HTML semantic elements. Can be merged.

**Review by Hamdrive — COMMENTED** [review]
Great work done for first PR, UI looks very clean
Good work implementing responsiveness as well
Will suggest to look into using a brighter text color, for better contrast and readability

**Comment by lannisterrr** [comment]
Got two reviews , merging the branch now.

**Pull request merged**